app/javascript/features/companies/manage/SquadBuilder.jsx

```diff
@@ -104,6 +112,12 @@ export const SquadBuilder = ({}) => {
 
   const isCompanyUnlocksChange = useSelector(selectIsCompanyUnlocksChanged)
 
+  const [openDelete, setOpenDelete] = React.useState(false)
+  const [openTransportDelete, setOpenTransportDelete] = React.useState(false)
+  
+  const [squad, setSquad] = React.useState(null)
```

Comment:
`squad` conflicts with other args named `squad` in methods below. Can we rename this to `squadToDelete`?